### DISCUSSION TITLE
Download speed limits

### DISCUSSION THREAD

**Discussion by tastymonkey** [Kalanyr/gogrepo#91] [Ideas]
While I have a decent download speed now days the script can saturate my download and cause my family grief, along with myself on regular browsing, while it is grabbing games. At the moment I cannot implement QoS on the firewall. Even if it is just something to put in the command to force it to a max download speed, that would be wonderful.

Thanks

**Comment by Kalanyr** [comment]
Let me try and explain better: this method will cap speed but to use a
metaphor it does so by checking if water is coming through the tap too fast
and turning the tap off if it is and doing this really quickly so while it
will approximately limit the speed to that set particularly over a long
period of time it will go both over and under in the short term depending
on the immediate use , so its not a good solution if you need a hard cap
but if you just want an approximate limit its fine ( this is fine with me
the download limit works on a similar principle). I just don't want to
mislead you into thinking that if you set a 10 MB/s limit that it will
always be under that, it won't.
